docs/secondary-navigation.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: sub-navigation
+order: 5
+title: Secondary navigation
+description: Allow users to navigate between pages scoped to a section.
+---
+
+{% from "example/macro.njk" import appExample %}
+
+{{ appExample({
+  example: "secondary-navigation"
+}) }}
+
+## When to use this component
+
+Use the secondary navigation when you already have a [primary navigation](primary-navigation) and need to let users navigate between pages that are scoped to a particular section or concept.
+
+For example, if your service is a case management system, you might use the secondary navigation if there are multiple pages associated with a single case.
+
+## When not to use this component
+
+Do not use this component for the primary sections of your service, or if a [sub-navigation](sub-navigation) would be more appropriate.
+
+You could also consider using the [tabs component](https://design-system.service.gov.uk/components/tabs/) if the amount of content in each sub-section is small.
+
+## How it works
+
+{{ appExample({
+  example: "secondary-navigation-with-header"
+}) }}
+
+Unlike [tabs](https://design-system.service.gov.uk/components/tabs/), secondary navigation does not use JavaScript. Each item within the navigation is a link to a separate full page.
+
+One of the items should always be highlighted as the current page. This item remains a link, but uses black text and has a line beneath it.
+
+The secondary navigation should appear before the `<main>` element of the page. This means that it is skipped when [skip link](https://design-system.service.gov.uk/components/skip-link/) is used.
+
+The scope of the secondary navigation can be given by a heading above it. When doing this, you should set `labelledBy` to be the id of the heading, so that it is associated with the heading for screen reader users.
```

Comment:
Avoid using description of visual placement (could be below, to one side…)

```suggestion
The scope of the secondary navigation can be given by an associated heading. When doing this, you should set `labelledBy` to be the `id` of the heading, so that it is associated with the heading for screen reader users.
```